lib/verify-stream.js

```diff
@@ -66,7 +66,7 @@ function jwsDecode(jwsSig, opts) {
   if (!header)
     return null;
 
-  var payload = payloadFromJWS(jwsSig);
+  var payload = payloadFromJWS(jwsSig, opts.encoding);
   if (header.typ === 'JWT' || opts.json)
     payload = JSON.parse(payload, opts.encoding);
```

Comment:
Remove opts.encoding as a second argument to `JSON.parse`, as this is an argument for the `reviver` function, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse: 
> Non-callable values are ignored.